.github/CODEOWNERS

```diff
@@ -132,12 +132,12 @@
 
 # PHP
 /lib                                                                 @timothybjacobs @spacedmonkey
-/lib/global-styles.php                                               @timothybjacobs @spacedmonkey @oandregal
-/lib/class-wp-rest-block-editor-settings-controller.php              @timothybjacobs @spacedmonkey @geriux
 /lib/compat/wordpress-5.9/theme.json                                 @timothybjacobs @spacedmonkey @oandregal
 /lib/compat/wordpress-5.9/theme-i18n.json                            @timothybjacobs @spacedmonkey @oandregal
 /lib/compat/wordpress-5.9/class-wp-theme-json-gutenberg.php          @timothybjacobs @spacedmonkey @oandregal
 /lib/compat/wordpress-5.9/class-wp-theme-json-resolver-gutenberg.php @timothybjacobs @spacedmonkey @oandregal
+/lib/compat/wordpress-6.0/global-styles.php                          @timothybjacobs @spacedmonkey @oandregal
```

Comment:
@oandregal, can you confirm that all changes in https://github.com/WordPress/gutenberg/blob/1db15f0072f02b6a327991c7789839d607c568f5/lib/compat/wordpress-6.0/global-styles.php should be backported to WordPress 6.0?

Reply:
Got this link https://github.com/WordPress/wordpress-develop/commit/872b81879415748b3177f73db5280c6348514c21 from WP 5.9. I will update the patch.

Reply:
Yeah, the remaning changes in `lib/global-styles.php` were all backported for 5.9. Given the remaining code in the file only deals with kses it can also renamed to `kses.php` for clarity.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/39972/commits/0a8e9c9a33577224a934fa9ddc6b064412744a86 👍🏻